app/controllers/test_controller.rb

```diff
@@ -118,6 +118,13 @@ def new_task
     render_success 'task', t
   end
 
+  def new_team_data_field()
+    Team.current = Team.find(params[:team_id])
+    team_data_field = "Team-#{params[:fieldset]}"+Time.now.to_s
+    tt = create_team_task({label:team_data_field, team_id: params[:team_id], fieldset: params[:fieldset]}.merge(params))
+    render_success 'task', tt
```

Comment:
Since we're creating a `TeamTask` and not a `Task`, it's better to return `team_task` here, no?